hack/dockerfile/install-binaries.sh

```diff
@@ -52,14 +52,29 @@ install_dockercli() {
 	echo "Install docker/cli version $DOCKERCLI_VERSION from $DOCKERCLI_CHANNEL"
 
 	arch=$(uname -m)
-	echo "Docker CLI ARCH $arch"
+	# No official release of these platforms
+	if [[ "$arch" == "ppc64le" ]]; then
```

Comment:
Can you do if not eq x86_64/armhf/s390x here instead? That should cover all the arm variants.